packs/equipment-effects/effect-necrotic-cap.json

```diff
@@ -0,0 +1,71 @@
+{
+    "_id": "wQDfwrrvfRxwG5CY",
+    "img": "icons/consumables/mushrooms/umbontae-bumpy-purple.webp",
+    "name": "Effect: Necrotic Cap",
+    "system": {
+        "description": {
+            "value": "<p>Granted by @UUID[Compendium.pf2e.equipment-srd.Item.Necrotic Cap (Lesser)]{Necrotic Cap}</p>\n<p>Your @UUID[Compendium.pf2e.spells-srd.Item.Acid Grip] spell loses the acid trait, gains the fungus trait, and all acid damage the spell deals becomes void damage.</p>"
```

Comment:
Consider linking to all variants of the Necrotic Cap. For an example, see Effect: Shining Symbol

Reply:
Consider it considered.